sdk/personalizer/azure-ai-personalizer/README.md

```diff
@@ -4,16 +4,47 @@
 is a cloud-based service that helps your applications choose the best content item to show your users. You can use the Personalizer service to determine what product to suggest to shoppers or to figure out the optimal position for an advertisement. After the content is shown to the user, your application monitors the user's reaction and reports a reward score back to the Personalizer service. This ensures continuous improvement of the machine learning model, and Personalizer's ability to select the best content item based on the contextual information it receives.
 
 ## Getting started
-TBD
+
+### Prerequisites
+- A [Java Development Kit (JDK)][jdk_link], version 8 or later.
+- [Azure Subscription][azure_subscription]
+- [Cognitive Services for Personalizer account][personalizer_account] to use this package.
 
 ### Install the package
-TBD
+Include the dependency in the dependencies' section. Ignoring the version tag without the version tag.
+
+```xml
+<dependencies>
```

Comment:
Need to add these script tags
[//]: # ({x-version-update-start;com.azure:azure-ai-personalizer;current})